geanyvc/src/geanyvc.c

```diff
@@ -1663,7 +1663,8 @@ create_commitDialog(void)
 
 	/* line/column status label */
 	lineColumnLabel = gtk_label_new("");
-	gtk_misc_set_alignment(GTK_MISC(lineColumnLabel), 0, 0.5);
+	gtk_widget_set_halign(GTK_WIDGET(lineColumnLabel), GTK_ALIGN_START);
+	gtk_widget_set_valign(GTK_WIDGET(lineColumnLabel), GTK_ALIGN_CENTER);
```

Comment:
I'd leave the default `valign` of `FILL` (i.e. not explicitly set it: what this was after is left-align the label)

```suggestion
```